actions/views/drafts.ts

```diff
@@ -28,15 +29,36 @@ type Draft = {
 }
 
 export function getDrafts(teamId: string) {
-    return async (dispatch: DispatchFunc) => {
-        let drafts: ServerDraft[] = [];
-        drafts = await Client4.getUserDrafts(teamId);
-        const actions = (drafts || []).map((draft) => {
-            const {key, value} = transformServerDraft(draft);
-            return setGlobalItem(key, value);
+    const getLocalDrafts = makeGetDrafts();
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState() as GlobalState;
+
+        let serverDrafts: Draft[] = [];
+        if (syncedDraftsAreAllowedAndEnabled(state)) {
+            serverDrafts = (await Client4.getUserDrafts(teamId) || []).map((draft) => transformServerDraft(draft));
```

Comment:
These existed with the previous PR so it's probably fine as-is, but I had a couple comments here:
1. We should probably have a try/catch around this client call so that, if an error occurs, it goes through the regular error handling stuff.
2. The `|| []` on the result of that seems a bit unnecessary to me. I'd expect that either this errors out entirely or that it would return a proper array